relay-server/src/actors/spooler/mod.rs

```diff
@@ -396,10 +425,19 @@ impl OnDisk {
             .try_get("received_at")
             .map_err(BufferError::FetchFailed)?;
         let start_time = StartTime::from_timestamp_millis(received_at as u64);
+        let own_key: &str = row.try_get("own_key").map_err(BufferError::FetchFailed)?;
+        let sampling_key: &str = row
+            .try_get("sampling_key")
```

Comment:
not really this PR, but thoughts on an enum that represents the columns? it could be a nice documentation for how the DB looks like. I assume try_get would accept it if it implements display

Reply:
maybe we can look into this in the followups 👍